<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1,0">
   <item>
    <widget class="QRadioButton" name="rbTranslateOp">
     <property name="text">
      <string>Add Translate Op</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbScaleOp">
     <property name="text">
      <string>Add Scale Op</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
     <item>
      <widget class="QRadioButton" name="rbRotateOp">
       <property name="text">
        <string>Add Rotate Op</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>- Order</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbOrder"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="rbOrientOp">
     <property name="toolTip">
      <string>arbitrary axis/angle rotation</string>
     </property>
     <property name="text">
      <string>Add Orient Op</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="rbTransform">
     <property name="text">
      <string>Add Transform Op</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Precision</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbPrecision"/>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="cbInvert">
       <property name="text">
        <string>Inverse-form</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="leSuffix"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Suffix</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbOK">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbRotateOp</sender>
   <signal>toggled(bool)</signal>
   <receiver>cbOrder</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
